platformio/src/main.cpp

```diff
@@ -16,6 +17,23 @@ int send_command(I2C& i2c, uint8_t address, uint16_t command) {
     return i2c.write(address, cmd, sizeof(cmd));
 }
 
+void ds18b20_send_command(OneWire& one_wire, uint8_t command) {

```

Comment:
Why are the `ds18b20` functions in `main.cpp`? I think a separate class would make more sense.